tutor/commands/upgrade/common.py

```diff
@@ -43,6 +43,46 @@ def upgrade_from_nutmeg(context: click.Context, config: Config) -> None:
     )
 
 
+def get_mysql_change_authentication_plugin_query(config: Config) -> str:
```

Comment:
Wondering if we should have an override/additional param provided by user to only upgrade a particular plugin's database. Lets say they have upgraded the default already and do not have plugins enabled. Once plugins are enabled, they will be re-doing the migration for default.

Reply:
Yes, I was also thinking that we could add a `--users` option with comma separated name of users to the do command. This way, users can also choose to upgrade their choice of databases.

Reply:
I've added a `--users` option as well.